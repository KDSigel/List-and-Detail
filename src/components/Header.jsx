import React from 'react'

export default function Header() {
    return (
        <div>
            <Link to='/'>character list</Link>
        </div>
    )
}
