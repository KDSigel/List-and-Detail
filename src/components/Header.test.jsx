import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import Header from './Header'

it('tests that the header shows up', () => {
const {container} = render(
    <MemoryRouter>
        <Header />
    </MemoryRouter>

)

expect(container).toMatchSnapshot()
}
)