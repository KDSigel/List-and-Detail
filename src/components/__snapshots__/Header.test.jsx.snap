// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests that the header shows up 1`] = `
<div>
  <div>
    <a
      href="/"
    >
      character list
    </a>
  </div>
</div>
`;
